docs/en/sql-reference/20-sql-functions/10-semi-structured-functions/json-object-keep-null.md

```diff
@@ -0,0 +1,66 @@
+---
+title: JSON_OBJECT_KEEP_NULL
+title_includes: TRY_JSON_OBJECT_KEEP_NULL
+---
+import FunctionDescription from '@site/src/components/FunctionDescription';
+
+<FunctionDescription description="Introduced or updated: v1.2.402"/>
+
+Creates a JSON object with keys and values.
+
+- The arguments are zero or more key-value pairs(where keys are strings, and values are of any type).
+- If the key is NULL, the key-value pair is ommitted from the resulting object. However, if the value is NULL, the key-value pair is keet.
+- The key cannot be duplicated and the order in which it was created cannot be maintained.
```

Comment:
The keys must be distinct from each other, and their order in the resulting JSON might be different from the order you specify.